python/py-virtualenv/Portfile

```diff
@@ -22,38 +22,31 @@ homepage            https://virtualenv.pypa.io
 master_sites        pypi:v/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  1ec5152c8d82350fd24358cfa0d112fdf70e7094 \
-                    sha256  0d62c70883c0342d59c11d0ddac0d954d0431321a41ab20851facf2b222598f3 \
-                    size    5121717
+checksums           rmd160  a3d2e5b06a9bfbc366571961966ccea70a74daca \
+                    sha256  7e4e234e1f27755685dc54063d989756790b0682d60e304db589fa1604938013 \
+                    size    7972183
 
 # keep older Python versions here, do add the EOL version to the list below
 python.versions     27 34 35 36 37 38
 
 if {${name} ne ${subport}} {
     # add EOL warnings
-    if {${python.version} in "34"} {
+    if {${python.version} in {"27" "34"}} {
         PortGroup   deprecated 1.0
         deprecated.eol_version  yes
     }
 
-    depends_lib-append  port:py${python.version}-setuptools
+     depends_lib-append     port:py${python.version}-appdirs \
+                            port:py${python.version}-distlib \
```

Comment:
you appear to be missing a few dependencies: 
- ```six``` for all Python versions
- ```contextlib2``` for Python < 3.3
- ```importlib-resources``` for Python < 3.7
- ```pathlib2``` for Python < 3.4

please check under ```install_requires``` in upstream's ```setup.cfg``` for the released version.

Reply:
additionally, you'll need to add subports for PY34 back to the new dependencies of ```virtualenv```

Reply:
Does `python34` need to be supported? I think it would be better to remove support for `python27` and support `python >= 3.5`.

Reply:
Yes, that decision was  made a long time ago (there have been comments about this on the mailinglist and/or other PRs, cannot find them right now). Port like ```virtualenv```, ```pip```, ```setuptools``` should keep supporting Python versions that are present in MacPorts (i.e., in lang/pythonXY) so that they can be used to install packages (from PyPI) even though these themselves are not present in MacPorts anymore.

Reply:
Very clear. I thought that `port destroot` would stop after destroot (and before post-destroot). 

Thanks!